spark/src/main/scala/ai/chronon/spark/JoinBase.scala

```diff
@@ -239,7 +236,7 @@ abstract class JoinBase(joinConf: api.Join,
     val rightBloomMap = if (skipBloom) {
       None
     } else {
-      JoinUtils.genBloomFilterIfNeeded(joinPart, joinConf, rowCount, unfilledRange, joinLevelBloomMapOpt)
+      JoinUtils.genBloomFilterIfNeeded(joinPart, joinConf, rowCount, unfilledRange, bloomMapOpt)
```

Comment:
nit: I think this function no longer generate BF but simply pull BFs for the corresponding key, so we can rename it